.github/ISSUE_TEMPLATE/operate-feature.md

```diff
@@ -6,38 +6,33 @@ labels: ["kind/feature", "component/operate"]
 assignees: ''
 
 ---
-<!-- Remember to link this feature issue in the parent Epic -->
 
-### Description
-<!-- [Optional] A clear description of the story -->
+### Description (required on creation)
+
+<!-- A clear description of the feature -->
+
+### User story (Required on creation)
 
-### User Story
-<!-- [Mandatory field] -->
 ```Gherkin
 Scenario: As an Operate user
     Given I am logged in
     When I select one item/tab in the toolbar
     Then this item should be shown in highlighted (in bold)
 ```
+
 ### Designs
+
 <!-- [Optional] <Figma Link> -->
 
-### Acceptance Criteria
-<!-- [Mandatory field] The assignee will fill the Acceptance Criteria. -->
-- [ ]
```

Comment:
I think these acceptance criteria made it easier for the QA engineers to know what exactly to test. Do you think we can keep it?

Reply:
In my opinion the user story plus functional and technical requirements should be the basis of what to test. So the acceptance criteria of a feature are that the requirements are met. Do you have an example of where those differed from explicity additional acceptance criteria (or if you can envision such a case more concretely)?

Generally I'm afraid that acceptance criteria, requirements and user story become redundant. I may not be clear what to put where (to me personally it wouldn't be clear; I'd need more education or docs to write this correctly).

Reply:
I agree that these would fit into the functional requirements section.